test/testfiles/en-US.properties

```diff
@@ -0,0 +1,15 @@
+resource1 = translation 1
+resource2 = translation 2
+
+# comment for resource3
+resource3 = translation 3
+
+resource4 = translation 4  # this is a comment for translation 4
```

Comment:
Looks like Java properties file does not support appending comments after a value https://en.wikipedia.org/wiki/.properties#Format

Reply:
removed